docs/pages/setup/guides/fluentd.mdx

```diff
@@ -74,6 +81,23 @@ Run the `configure` command to generate a sample configuration. Replace
 $ ./teleport-event-handler configure . mytenant.teleport.sh
 ```
 
+</TabItem>
+<TabItem scope={["oss", "enterprise", "cloud"]} label="Helm Chart">
+
+Run the `configure` command to generate a sample configuration. Replace `mytenant.teleport.sh` with the DNS name of your Teleport server (Teleport Cloud or self-hosted):
```

Comment:
```suggestion
Run the `configure` command to generate a sample configuration. Assign `TELEPORT_CLUSTER_ADDRESS` to the DNS name and port of your Teleport Auth Service or Proxy Service:
```
Suggesting using an env variable since `mytenant.teleport.sh` is not visible unless you scroll